t/26-ssh_screen.t

```diff
@@ -21,12 +22,30 @@ is($screen->{fd_write}, 'My_Chan', 'SSH channel is used for writing');
 subtest 'Correct message when type_string timeouts' => sub {
     my $mock_screenconsole = Test::MockModule->new('consoles::serial_screen');
     $mock_screenconsole->mock('elapsed', sub { 1000 });
-    my $mock_ssh = Test::MockObject->new();
+    my $mock_connection = Test::MockObject->new();
     my $mock_channel = Test::MockObject->new();
     my $mock_bmwqemu = Test::MockModule->new('bmwqemu');
     $mock_bmwqemu->noop('log_call');
-    my $sshscreen = consoles::ssh_screen->new(ssh_connection => $mock_ssh, ssh_channel => $mock_channel);
+    my $sshscreen = consoles::ssh_screen->new(ssh_connection => $mock_connection, ssh_channel => $mock_channel);
     throws_ok { $sshscreen->type_string({text => 'This should timeout'}) } qr/consoles::ssh_screen::type_string: Timed out after 1000 seconds/, "sub dies with correct error message and display the correct caller";
 };
 
+subtest 'test old net ssh2 error handling' => sub {
+    my $mock_screenconsole = Test::MockModule->new('consoles::serial_screen');
+    my $mock_connection = Test::MockObject->new();
+    my $mock_write_attempts = 0;
+    my $mock_channel = Test::MockObject->new();
+    $mock_channel->mock('write', sub {
+            $mock_write_attempts++;
+            return -37 if $mock_write_attempts <= 3;
```

Comment:
What's this -37? Can we have the magic number here?

Reply:
You can just use the constant exported by the module:
```suggestion
            return LIBSSH2_ERROR_EAGAIN if $mock_write_attempts <= 3;
```